pkgs/top-level/go-packages.nix

```diff
@@ -1230,6 +1230,30 @@ let
     };
   };
 
+  go-runewidth = buildGoPackage rec {
+    rev = "travisish";
+    name = "go-runewidth-${rev}";
+    goPackagePath = "github.com/junegunn/go-runewidth";
```

Comment:
This is actually forked from https://github.com/mattn/go-runewidth, so maybe I should put the junegunn version under a different name?